syntax = "proto3";

import "field_size.proto";

package test_messages;

message TestString {
  string value = 1 [(field_size) = 20];
}

message TestRepeatedString {
  repeated string value = 1 [(field_size) = 20];
}

message TestUint32 {
  uint32 value = 1;
}

message TestNestedMessage{
  TestUint32 message = 1;
}
